service/k8sclient/k8sclient.go

```diff
@@ -69,18 +71,18 @@ const (
 	k8sAPIVersion     = "v1"
 	k8sMetaKindSecret = "Secret"
 
-	pxcCRVersion         = "1.7.0"
-	pxcBackupImage       = "percona/percona-xtradb-cluster-operator:1.6.0-pxc8.0-backup"
-	pxcImage             = "percona/percona-xtradb-cluster:8.0.20-11.1"
-	pxcBackupStorageName = "pxc-backup-storage-%s"
-	pxcAPIVersion        = "pxc.percona.com/v1-6-0"
-	pxcProxySQLImage     = "percona/percona-xtradb-cluster-operator:1.6.0-proxysql"
-	pxcSecretNameTmpl    = "dbaas-%s-pxc-secrets"
-	defaultPXCSecretName = "my-cluster-secrets"
+	pxcCRVersion            = "1.7.0"
+	pxcBackupImage          = "percona/percona-xtradb-cluster-operator:1.6.0-pxc8.0-backup"
+	pxcDefaultImage         = "percona/percona-xtradb-cluster:8.0.20-11.1"
+	pxcBackupStorageName    = "pxc-backup-storage-%s"
+	pxcAPIVersion           = "pxc.percona.com/v1-7-0"
+	pxcProxySQLDefaultImage = "percona/percona-xtradb-cluster-operator:1.6.0-proxysql"
```

Comment:
@BupycHuk Why do we use 1.7.0 as `pxcAPIVersion` and `pxcCRVersion`, and 1.6.0 as version for `pxcBackupImage` and `pxcProxySQLDefaultImage`?

Reply:
Good point